### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by Bradben**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/azure-quantum-docs-issue.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+about: Template for documentation related issues
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- For issues related to the documentation use this repository.
+- For technical issues with Azure Quantum and optimization please use [Azure Support](https://ms.portal.azure.com/#blade/Microsoft_Azure_Support/HelpAndSupportBlade/newsupportrequest) or [Azure Q&A](https://docs.microsoft.com/en-us/answers/topics/azure-quantum.html).
+- For feature asks around Azure Quantum please use the [feedback forum](https://feedback.azure.com/forums/932905-azure-quantum).
```
Comment (dime10) [inline .github/ISSUE_TEMPLATE/azure-quantum-docs-issue.md:12]: Maybe feature request is the more commonly used term here.
```suggestion
- For feature requests around Azure Quantum please use the [feedback forum](https://feedback.azure.com/forums/932905-azure-quantum).
```

File: .github/ISSUE_TEMPLATE/azure-quantum-docs-issue.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- Q# and QDK continue to be supported via their GitHub repos.
```
Comment (dime10) [inline .github/ISSUE_TEMPLATE/azure-quantum-docs-issue.md:13]:
```suggestion
- Q# and the QDK continue to be supported via their GitHub repos.
```

**Review by dime10 — APPROVED** [review]
Thanks Brad!

File: .github/ISSUE_TEMPLATE/azure-quantum-docs-issue.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- For issues related to the documentation use this repository.
+- For technical issues with Azure Quantum and optimization please use [Azure Support](https://ms.portal.azure.com/#blade/Microsoft_Azure_Support/HelpAndSupportBlade/newsupportrequest) or [Azure Q&A](https://docs.microsoft.com/en-us/answers/topics/azure-quantum.html).
+- For feature requests around Azure Quantum please use the [feedback forum](https://feedback.azure.com/forums/932905-azure-quantum).
+- Q# and the QDK continue to be supported via their GitHub repos.
```
Comment (Bradben) [inline .github/ISSUE_TEMPLATE/azure-quantum-docs-issue.md:13]:
```suggestion
- Q# and the QDK continue to be supported via their GitHub repos: [Q# language](https://github.com/microsoft/qsharp-language/issues) and [QDK](https://github.com/microsoft/qsharp-runtime/issues).
```

**Pull request merged**